src/08-tasks/tasks.apib

```diff
@@ -59,7 +59,10 @@ Create a new task.
 
     + Attributes (object)
         + description (string, required)
-        + due_at: `2016-02-04T16:44:33+00:00` (string, optional)
+        + due_at: `2016-02-04T16:44:33+00:00` (string, required)
+        + work_type_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, required)
+        + estimated_duration: 3600 (number, optional)
```

Comment:
Is it clear enough that this is in seconds? 3600 is a big hint of course

Reply:
Damn, we did that in #294 but forgot to do it here. Fixing